modules/library/src/test/scala/zio/elasticsearch/SttpBackendStubSpec.scala

```diff
@@ -0,0 +1,216 @@
+package zio.elasticsearch
+
+import sttp.client3.httpclient.zio.HttpClientZioBackend
+import sttp.client3.testing.SttpBackendStub
+import sttp.client3.{Request, Response, StringBody}
+import sttp.model.{Method, StatusCode}
+import zio.test.ZIOSpecDefault
+import zio.{Task, TaskLayer, ZLayer}
+
+trait SttpBackendStubSpec extends ZIOSpecDefault {
+
+  case class StubMapping(request: Request[_, _] => Boolean, response: Response[Any])
+
+  final implicit class SttpBackendStubImplicits[T](sttpBackendStub: SttpBackendStub[Task, T]) {
```

Comment:
Does it make sense to use `SttpBackendStubOps` convention? Also, let's use `[A]`, instead of `[T]`. I always feel like `T` is Java convention/leftover. 😄

Reply:
It makes totally sense !